packages/core/jest.config.js

```diff
@@ -2,5 +2,6 @@ const { jest: lernaAliases } = require('lerna-alias');
 
 module.exports = {
   preset: 'ts-jest',
-  moduleNameMapper: lernaAliases()
+  moduleNameMapper: lernaAliases(),
+  watchPathIgnorePatterns: ['/lib/', '/dist/', '/es/']
```

Comment:
does this bring any noticeable gains? we already have a very different setup on the next branch (which simplifies stuff quite a bit automatically~) so if those changes to `jest.config.js` files are not necessarily then I would like to avoid those to avoid merge conflicts etc

Reply:
In `yarn dev`, changing a file used to cause the jest tests to run two+ times for each single change. This fixes it.